src/ScaffoldInstallerPlugin.php

```diff
@@ -326,23 +332,61 @@ private function installGitlabCI(string $scaffoldPath): void {
                 if (!file_exists('./pantheon.yml')) {
                     $fs->copy("$scaffoldPath/pantheon/pantheon.yml", './pantheon.yml');
                 }
-                // settings.pantheon.php
-                if (!file_exists('./web/sites/default/settings.pantheon.php')) {
-                    $fs->copy("$scaffoldPath/pantheon/settings.pantheon.php", './web/sites/default/settings.pantheon.php');
-                }
+
+                $this->checkPantheonSystemDrupalIntegrations($composer);
             }
         }
         if (!file_exists('./.gitlab-ci.yml')) {
             $fs->copy("$scaffoldPath/gitlab/gitlab-ci.example.yml", './.gitlab-ci.yml');
         }
     }
 
+    /**
+     * Check if package pantheon-systems/drupal-integrations is installed.
+     * If not installed, recommend the user to install it.
+     *
+     * @param \Composer\Composer $composer
+     *   The Composer instance.
+     *
+     * @return void
+     *   No return value.
+     */
+    private function checkPantheonSystemDrupalIntegrations(Composer $composer): void {
+        $repositoryManager = $composer->getRepositoryManager();
+        $localRepository = $repositoryManager->getLocalRepository();
+        $package = $localRepository->findPackage('pantheon-systems/drupal-integrations', '*');
+        if ($package) {
+            return; // Found the package, no warning needed.
+        }
+        $this->pantheonSystemDrupalIntegrationsWarning();
+    }
+
+    /**
+     * Check for common Pantheon configuration files.
+     *
+     * @return bool
+     *   True if the site uses Pantheon, false otherwise.
+     */
+    private function hasPantheonConfigurationFiles(): bool
+    {
+        return file_exists('./pantheon.yml')
+            || file_exists('./pantheon.upstream.yml')
+            || file_exists('./.pantheon');
+    }
+
+    /**
+     * Display a warning about the pantheon-systems/drupal-integrations package.
+     */
+    private function pantheonSystemDrupalIntegrationsWarning(): void {
```

Comment:
Could this method just be moved inside checkPantheonSystemDrupalIntegrations()? It's only called from that method and not by anything else.

Reply:
I checked and this is called 3 times in src/ScaffoldInstallerPlugin.php